03-Binary-Search-Trees/03-Binary-Search-Trees-Height-Big-O.md

```diff
@@ -474,6 +474,11 @@ If the current node is nil return 0
 
 Otherwise return 1 plus the maximum of the heights of the right and left subtrees
 ```
+
+Still feeling stuck? Check this video walkthrough of the solution.
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=996e137b-e4f2-460e-a2e8-af0e01521de6&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="360" width="640" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
```

Comment:
- Really loved the whiteboard walkthrough here. Consider taking it a step further and directly pointing  out that to find the max height of a subtree, just treat the subtree as its own separate tree where the child becomes the root. Eventually, when you reach a leaf it's as if you had the first case you initially talked about where the tree is just the root.
- Consider making a more direct callout that the `1` you add to the result of `max(height(left), height(right))` accounts for the current root or level you are looking at